From 89431247598c378cba6b85d010135317375905eb Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 24 May 2020 21:14:47 +0000
Subject: [PATCH 02/18] Relax constraint on minimum version of GCC

---
 SConstruct | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SConstruct b/SConstruct
index aae4f1132dd..2a37ecfdc1c 100644
--- a/SConstruct
+++ b/SConstruct
@@ -2683,14 +2683,14 @@ def doConfigure(myenv):
             }
             """ % compiler_minimum_string)
         else:
-            compiler_minimum_string = "GCC 8.2"
+            compiler_minimum_string = "GCC 7.5"
             compiler_test_body = textwrap.dedent(
             """
             #if !defined(__GNUC__) || defined(__clang__)
             #error
             #endif
 
-            #if (__GNUC__ < 8) || (__GNUC__ == 8 && __GNUC_MINOR__ < 2)
+            #if (__GNUC__ < 7) || (__GNUC__ == 7 && __GNUC_MINOR__ < 5)
             #error %s or newer is required to build MongoDB
             #endif
 
-- 
2.33.1

