From 7331c0c1f74a184df9b2f1556c2d21ffa8c0c178 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 15:22:50 +0000
Subject: [PATCH 03/17] Make SCons inherit environment vars

---
 SConstruct | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SConstruct b/SConstruct
index 98fe5bce85d..144e2babe05 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1651,6 +1651,7 @@ if get_option('build-tools') == 'next':
     SCons.Tool.DefaultToolpath.insert(0, os.path.abspath('site_scons/site_tools/next'))
 
 env = Environment(variables=env_vars, **envDict)
+env.Append(ENV = os.environ)
 del envDict
 env.AddMethod(lambda env, name, **kwargs: add_option(name, **kwargs), 'AddOption')
 
