From f922a640b8fa0c76e5bda76339aab0e8e45c9502 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 15:22:50 +0000
Subject: [PATCH 06/19] Make SCons inherit environment vars

---
 SConstruct | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SConstruct b/SConstruct
index d9f20873e6a..36ffd86d8f6 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1220,6 +1220,7 @@ if get_option('build-tools') == 'next':
     SCons.Tool.DefaultToolpath.insert(0, os.path.abspath('site_scons/site_tools/next'))
 
 env = Environment(variables=env_vars, **envDict)
+env.Append(ENV = os.environ)
 del envDict
 
 if get_option('cache-signature-mode') == 'validate':
-- 
2.33.0

