From 2ede7bdbfe94e41cd4de32aea1b438f1f9d1fad2 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 16:48:04 +0000
Subject: [PATCH 06/17] Relax minimum Darwin version check to 10.9

---
 SConstruct | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SConstruct b/SConstruct
index 5a6434e9070..39079a29d41 100644
--- a/SConstruct
+++ b/SConstruct
@@ -3406,7 +3406,7 @@ def doConfigure(myenv):
 
     conf.Finish()
 
-    # We require macOS 10.14 or newer
+    # We require macOS 10.9 or newer
     if env.TargetOSIs('darwin'):
 
         # TODO: Better error messages, mention the various -mX-version-min-flags in the error, and
@@ -3417,7 +3417,7 @@ def doConfigure(myenv):
             #include <AvailabilityMacros.h>
             #include <TargetConditionals.h>
 
-            #if TARGET_OS_OSX && (__MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_14)
+            #if TARGET_OS_OSX && (__MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_9)
             #error 1
             #endif
             """
@@ -3437,7 +3437,7 @@ def doConfigure(myenv):
         )
 
         if not conf.CheckDarwinMinima():
-            conf.env.ConfError("Required target minimum of macOS 10.14 not found")
+            conf.env.ConfError("Required target minimum of macOS 10.9 not found")
 
         conf.Finish()
 
