From d328b4183f8069541aad456b52734a08f8e3a302 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 16:48:04 +0000
Subject: [PATCH 08/17] Relax minimum Darwin version check to 10.9

---
 SConstruct | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/SConstruct b/SConstruct
index 031c406fa0..46c5560e40 100644
--- a/SConstruct
+++ b/SConstruct
@@ -2703,7 +2703,7 @@ def doConfigure(myenv):
 
     conf.Finish()
 
-    # We require macOS 10.14 or newer
+    # We require macOS 10.9 or newer
     if env.TargetOSIs('darwin'):
 
         # TODO: Better error messages, mention the various -mX-version-min-flags in the error, and
@@ -2714,7 +2714,7 @@ def doConfigure(myenv):
             #include <AvailabilityMacros.h>
             #include <TargetConditionals.h>
 
-            #if TARGET_OS_OSX && (__MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_14)
+            #if TARGET_OS_OSX && (__MAC_OS_X_VERSION_MIN_REQUIRED < __MAC_10_9)
             #error 1
             #endif
             """
@@ -2729,7 +2729,7 @@ def doConfigure(myenv):
         })
 
         if not conf.CheckDarwinMinima():
-            conf.env.ConfError("Required target minimum of macOS 10.14 not found")
+            conf.env.ConfError("Required target minimum of macOS 10.9 not found")
 
         conf.Finish()
 
-- 
2.32.0

