From 768d0ec9d0caf5fb661c989a3473598cadb564e6 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 19:21:32 +0000
Subject: [PATCH 09/21] Use platform specific TimeStamp implementation

---
 src/third_party/mozjs/SConscript | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/src/third_party/mozjs/SConscript b/src/third_party/mozjs/SConscript
index b649e28abba..aa9f1f1e43b 100644
--- a/src/third_party/mozjs/SConscript
+++ b/src/third_party/mozjs/SConscript
@@ -146,9 +146,15 @@ else:
     sources.extend([
     "extract/mozglue/misc/ConditionVariable_posix.cpp",
     "extract/mozglue/misc/Mutex_posix.cpp",
-    "extract/mozglue/misc/TimeStamp_posix.cpp",
     ])
-
+    if env.TargetOSIs('darwin'):
+        sources.extend([
+            "extract/mozglue/misc/TimeStamp_darwin.cpp"
+        ])
+    else:
+        sources.extend([
+            "extract/mozglue/misc/TimeStamp_posix.cpp"
+        ])
 
 
 sources.append( [ "extract/modules/fdlibm/{}".format(f) for f in [
