From 0b8c27d3830dc50c2ff9ebf739b45621f1579abc Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sat, 29 Jan 2022 17:43:42 -0500
Subject: [PATCH 10/15] Drop strict python pkgs requirements check

---
 buildscripts/scons.py | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/buildscripts/scons.py b/buildscripts/scons.py
index b1ddc85e54d..8befd0244e9 100755
--- a/buildscripts/scons.py
+++ b/buildscripts/scons.py
@@ -18,15 +18,6 @@ SITE_TOOLS_DIR = os.path.join(MONGODB_ROOT, 'site_scons')
 
 sys.path = [SCONS_DIR, SITE_TOOLS_DIR] + sys.path
 
-# pylint: disable=C0413
-from mongo.pip_requirements import verify_requirements, MissingRequirements
-
-try:
-    verify_requirements()
-except MissingRequirements as ex:
-    print(ex)
-    sys.exit(1)
-
 try:
     import SCons.Script
 except ImportError as import_err:
