From 4170866d12588b91cd8ddddc32cc50af5f5d1833 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 10/18] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index fa319073e68..402701a320d 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -32,7 +32,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'resmoke',
