From bb4ca965c6bf1e9b82c0295d4f4aab00c497ca9a Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 13/18] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index 37e4ae0199c..93a5e795094 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -30,7 +30,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'resmoke',
