From 6774cc636b20b41c02093807743105ac4fe27851 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 25 Jul 2024 12:05:10 +1100
Subject: [PATCH 14/15] add missing include for abi::__cxa_demangle

---
 src/mongo/util/heap_profiler.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/mongo/util/heap_profiler.cpp b/src/mongo/util/heap_profiler.cpp
index 6af1279826d..6097f35b06c 100644
--- a/src/mongo/util/heap_profiler.cpp
+++ b/src/mongo/util/heap_profiler.cpp
@@ -73,6 +73,12 @@
 // for dlfcn.h and backtrace
 #if defined(_POSIX_VERSION) && defined(MONGO_CONFIG_HAVE_EXECINFO_BACKTRACE)
 
+#ifdef __linux__
+// for using abi::__cxa_demangle, see
+// https://gcc.gnu.org/onlinedocs/libstdc++/manual/ext_demangling.html
+#include <cxxabi.h>
+#endif
+
 //
 // Sampling heap profiler
 //
