From 21106931c30d5c296906b13cf8087460e11fb031 Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 25 Jul 2024 12:05:10 +1100
Subject: [PATCH 14/18] add missing include for abi::__cxa_demangle

---
 src/mongo/util/heap_profiler.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/mongo/util/heap_profiler.cpp b/src/mongo/util/heap_profiler.cpp
index 59029330dad..7ecc6a3631d 100644
--- a/src/mongo/util/heap_profiler.cpp
+++ b/src/mongo/util/heap_profiler.cpp
@@ -53,6 +53,12 @@
 // for dlfcn.h and backtrace
 #if defined(_POSIX_VERSION) && defined(MONGO_CONFIG_HAVE_EXECINFO_BACKTRACE)
 
+#ifdef __linux__
+// for using abi::__cxa_demangle, see
+// https://gcc.gnu.org/onlinedocs/libstdc++/manual/ext_demangling.html
+#include <cxxabi.h>
+#endif
+
 //
 // Sampling heap profiler
 //
