From 2551127ca615bf380c714c41ace5352c06b40424 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 15/20] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index 8696da79339..e6902ef3c21 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -32,7 +32,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'resmoke',
-- 
2.33.1

