From 99c41806866b5314e7cbfa6448cace4080e6adef Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Thu, 25 Jul 2024 12:05:10 +1100
Subject: [PATCH 16/19] add missing include for abi::__cxa_demangle

---
 src/mongo/util/heap_profiler.cpp | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/mongo/util/heap_profiler.cpp b/src/mongo/util/heap_profiler.cpp
index a7e3988392b..bfa55f41b8e 100644
--- a/src/mongo/util/heap_profiler.cpp
+++ b/src/mongo/util/heap_profiler.cpp
@@ -51,6 +51,12 @@
 #include <dlfcn.h>
 #include <execinfo.h>
 
+#ifdef __linux__
+// for using abi::__cxa_demangle, see
+// https://gcc.gnu.org/onlinedocs/libstdc++/manual/ext_demangling.html
+#include <cxxabi.h>
+#endif
+
 //
 // Sampling heap profiler
 //
