From 5f58d8fe575d5b409018f3dbd649ab5d7afd1894 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 17/17] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index d2925d1f0e..9b74ecff12 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -59,7 +59,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'rpc',
-- 
2.31.1

