From 19c9dc841b3f40a6b30fa441a64b65eb43276fea Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 17/17] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index 8a7e3af3a0..3c2f5f8563 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -32,7 +32,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'rpc',
-- 
2.33.1

