From d2bf97d6a2f5423d57eb70a66cc16293492d5c4a Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 11 Jul 2021 23:31:20 -0400
Subject: [PATCH 17/18] Disable installer on Windows

---
 src/mongo/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mongo/SConscript b/src/mongo/SConscript
index 4e4ec6aad5..667984dcf8 100644
--- a/src/mongo/SConscript
+++ b/src/mongo/SConscript
@@ -32,7 +32,7 @@ env.SConscript(
         'embedded',
         'executor',
         'idl',
-        'installer',
+#        'installer',
         'logv2',
         'platform',
         'rpc',
-- 
2.34.1

