From f7da8ac3bbe17c926cced5e7b292a8b4c19ff33f Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 17 Feb 2022 09:45:16 +0100
Subject: [PATCH 18/18] wyhash is now called low_level_hash

---
 src/third_party/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/third_party/SConscript b/src/third_party/SConscript
index 6d1333a62b8..2fd297b9277 100644
--- a/src/third_party/SConscript
+++ b/src/third_party/SConscript
@@ -383,7 +383,7 @@ if use_system_version_of_library("abseil-cpp"):
             #    absl::raw_hash_set
             #    absl::city absl::hash absl::wyhash
             #    absl::numeric)
-            'absl_raw_hash_set', 'absl_hash', 'absl_wyhash',
+            'absl_raw_hash_set', 'absl_hash', 'absl_low_level_hash',
             'absl_hashtablez_sampler', 'absl_exponential_biased',
             'absl_synchronization', 'absl_graphcycles_internal',
             'absl_stacktrace', 'absl_symbolize', 'absl_malloc_internal',
-- 
2.33.1

