From eebe38eab1334a1b271b42d7fa45df232e922ee7 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Mon, 25 May 2020 02:26:17 +0000
Subject: [PATCH 18/19] Fix flags for un-vendoring abseil-cpp

---
 src/third_party/SConscript | 18 +++++++++++++++---
 1 file changed, 15 insertions(+), 3 deletions(-)

diff --git a/src/third_party/SConscript b/src/third_party/SConscript
index c8b4716b124..5d4a86886c1 100644
--- a/src/third_party/SConscript
+++ b/src/third_party/SConscript
@@ -377,9 +377,21 @@ abseilEnv = env.Clone()
 if use_system_version_of_library("abseil-cpp"):
     abseilEnv = abseilEnv.Clone(
         SYSLIBDEPS=[
-            env['LIBDEPS_ABSL_CONTAINER_SYSLIBDEP'],
-            env['LIBDEPS_ABSL_HASH_SYSLIBDEP'],
-            env['LIBDEPS_ABSL_NUMERIC_SYSLIBDEP'],
+            # src/third_party/abseil-cpp-master/SConscript
+            # target_link_libraries(...
+            #    absl::raw_hash_set
+            #    absl::city absl::hash absl::wyhash
+            #    absl::numeric)
+            'absl_raw_hash_set', 'absl_hash', 'absl_wyhash',
+            'absl_hashtablez_sampler', 'absl_exponential_biased',
+            'absl_synchronization', 'absl_graphcycles_internal',
+            'absl_stacktrace', 'absl_symbolize', 'absl_malloc_internal',
+            'absl_debugging_internal', 'absl_demangle_internal', 'absl_time',
+            'absl_civil_time', 'absl_time_zone', 'absl_city',
+            'absl_bad_optional_access', 'absl_strings', 'absl_throw_delegate',
+            'absl_strings_internal', 'absl_base', 'absl_spinlock_wait',
+            'absl_bad_variant_access', 'absl_raw_logging_internal',
+            'absl_log_severity', 'absl_int128'
         ])
 else:
     abseilDirectory = 'abseil-cpp-master'
-- 
2.33.0

